README.md

```diff
@@ -69,7 +69,7 @@ This will walk you through running Atlantis locally. It will
 - install ngrok so we can expose Atlantis to GitHub
 - start Atlantis
 
-If you're ready to permanently set up Atlantis see [Production-Ready Deployment](#production-ready-deployment)
+If you're ready to permanently set up Atlantis see [Production-Ready Deployment](#production-ready-deployment) and check out [repository terraform-aws-atlantis](https://github.com/terraform-aws-modules/terraform-aws-atlantis) where complete Terraform configurations for running Atlantis on AWS Fargate are hosted.
```

Comment:
Instead of putting this here, can you create a new heading under https://github.com/runatlantis/atlantis#kubernetes after the words "You could also set up SSL at your LoadBalancer.".

The heading can be "AWS Fargate", ex:

### AWS Fargate
If you'd like to run Atlantis on [AWS Fargate](https://aws.amazon.com/fargate/) check out the Atlantis module on the Terraform Module Registry: https://registry.terraform.io/modules/terraform-aws-modules/atlantis/aws

---

Also add this section to the table of contents under the Kubernetes link.